lnbits/extensions/boltz/templates/boltz/_api_docs.html

```diff
@@ -73,21 +83,30 @@ <h3 class="text-subtitle1 q-my-none">
     :content-inset-level="0.5"
   >
     <q-card-section>
-      You want to swap in 100.000 sats, Onchain to Lightning:
+      {% raw %} You want to swap in {{ boltzExample.amount }} sats, Onchain to
+      Lightning:
       <ul style="padding-left: 12px">
         <li>Onchain lockup tx fee: whatever you choose when paying</li>
-        <li>Onchain claim tx fee: ~500 sats</li>
+        <li>Onchain claim tx fee: ~{{ boltzExample.onchain_boltz }} sats</li>
         <li>Routing fees (paid by boltz): unknown</li>
-        <li>Boltz fees: 500 sats (0.5%)</li>
-        <li>Fees total: 1000 sats + lockup_fee</li>
-        <li>You pay onchain: 101.000 sats + lockup_fee</li>
-        <li>You receive lightning: 100.000 sats</li>
+        <li>
+          Boltz fees: {{ boltzExample.boltz_fee }} sats ({{
+          boltzConfig.fee_percentage }}%)
+        </li>
+        <li>
+          Fees total: {{ boltzExample.normal_fee_total }} sats + lockup_fee
+        </li>
+        <li>
+          You pay onchain: {{ boltzExample.normal_expected_amount }} sats +
+          lockup_fee
+        </li>
+        <li>You receive lightning: {{ boltzExample.amount }} sats</li>
       </ul>
       <p>
         onchain_payment + lockup_fee = amount + (amount * boltz_fee / 100) +
         claim_fee + lockup_fee
```

Comment:
I'd remove the `+ lockup_fee` from both sides

Reply:
1c3a9d18